airsonic-main/src/main/java/org/airsonic/player/service/MediaScannerService.java

```diff
@@ -276,6 +276,12 @@ private void doScanLibrary(ForkJoinPool pool) {
 
             LOG.info("Completed media library scan.");
 
+            if (settingsService.isIgnoreFileTimestamps()) {
+                settingsService.setIgnoreFileTimestamps(false);
+                settingsService.save();
+                LOG.info("Ignore file timestamp property (for full scan) reset");
+            }
```

Comment:
Hmmm, there is a distinction to be made here between a one-time setting and a setting that stays scan-after-scan.
You're overriding it here.

Say a person wants to do a full scan every time, and sets the property in his prop file. This will override it (set it to false).